src/main/java/org/zone/commands/structure/misc/Messages.java

```diff
@@ -0,0 +1,295 @@
+        return Messages.getZonePluginInfoCommandPluginName(zone.getName());
+    }
+
+    public static Component getZoneInfoCommandZoneMembers(Collection<UUID> members) {
+        return Component.text("Members: " + members.size()).color(NamedTextColor.AQUA);
+    }
+
+    public static Component getZonePluginInfoCommandPluginName(String pluginName) {
+        return Component.text("Name: " + pluginName).color(NamedTextColor.AQUA);
+    }
+
+    public static Component getZonePluginInfoCommandPluginVersion(String pluginVersion) {
+        return Component.text("Version: " + pluginVersion).color(NamedTextColor.AQUA);
+    }
+
+    public static Component getZonePluginInfoCommandPluginGithub(String pluginGithub) {
+        return Component.text("Github: " + pluginGithub).color(NamedTextColor.AQUA);
+    }
+
+    public static Component getZonesPluginInfoCommandZonesNumber(String pluginZonesNumber) {
+        return Component.text("Zones: " + pluginZonesNumber).color(NamedTextColor.AQUA);
+    }
+
+    public static Component getZoneFlagLeavingSetMessageCommandIfNoLeavingFlagFound() {
+        return Component.text("Enter leaving message").color(NamedTextColor.RED);
+    }
+
+    public static Component getZoneFlagLeavingSetMessageCommandLeavingMessageSaved(Component message) {
+        return Component.text("Leaving message is now ").append(message).color(NamedTextColor.AQUA);
+    }
+
+    public static Component getZoneFlagLeavingMessageViewCommandMessageViewMessage(LeavingFlag leavingFlag) {
+        return Component
+                .text("Message: ")
+                .append(leavingFlag.getLeavingMessage()).color(NamedTextColor.AQUA);
+    }
+
+    public static Component getZoneFlagViewBalanceCommandBalanceMessage(@SuppressWarnings("TypeMayBeWeakened") Zone zone, BigDecimal decimal) {
+        return Component
+                .text(zone.getName())
+                .color(NamedTextColor.AQUA)
+                .append(Component
+                                .text(" balance: ")
+                                .color(NamedTextColor.AQUA)
+                                .append(Component
+                                                .text(decimal.toString())
+                                                .color(NamedTextColor.GOLD)));
+    }
+
+    public static Component getZoneFlagGreetingsSetMesssageCommandSetMessage(Component message) {
+        return Component
+                .text("Zone greetings message set to: ")
+                .append(message).color(NamedTextColor.AQUA);
+    }
+
+    public static Component getZoneFlagGreetingsRemoveCommandGreetingsRemovedMessage() {
+        return Component.text("Zone greetings message removed from this " +
+                                      "zone!").color(NamedTextColor.AQUA);
+    }
+
+    public static Component getZoneFlagBlockBreakSetGroupCommandGroupUpdatedFlagSaved() {
+        return Component.text("Updated Block Break Interaction " + "group").color(NamedTextColor.AQUA);
```

Comment:
Remove this. Use the generic edition created above

Reply:
errrr wbt the group then @mosemister